app/models/setting.rb

```diff
@@ -48,8 +48,16 @@ def validate(record)
   after_find :readonly_when_overridden_in_SETTINGS
   default_scope -> { order(:name) }
 
-  # The DB may contain settings from disabled plugins - filter them out here
-  scope :live_descendants, -> { where(:category => self.descendants.map(&:to_s)) unless Rails.env.development? }
+  # Filer out settings from disabled plugins and ones releated to taxonomies when required
```

Comment:
@sean797 You can use `Taxonomy.enabled_taxonomies` to find out which ones are enabled and build the query consequently. This is also missing some tests if you can add them? Thank you!